content/blog/2016/2016-05-10-jenkins-20-vjam.adoc

```diff
@@ -347,8 +347,8 @@ link:https://wiki.jenkins.io/display/JENKINS/Ownership+Plugin[Ownership]
 plugin may be of interest though.
 
 ____
-Please consider replacing terms like "master" and "slave" with "controller" and
-"agent".
+Please consider replacing terms like "master" and "slave" with "primary" and
+"secondary".
 ____
 
 "slave" has been replaced with "agent" in Jenkins 2.0
```

Comment:
If we want to add recent developments to this blog post, we could add something like

```suggestion
"slave" has been replaced with "agent" in Jenkins 2.0.

*Updated 2020-09-18*: The term "master" is being replaced with "controller".
```